mysql-test/include/have_rocksdb_as_default.inc

```diff
@@ -0,0 +1,18 @@
+--source include/have_rocksdb.inc
+
+if (`SELECT COUNT(*) = 0 FROM INFORMATION_SCHEMA.ENGINES WHERE engine = 'rocksdb' AND support IN ('DEFAULT')`)
+{
+  --skip Test requires default engine RocksDB
+}
+
+# MYR-14
```

Comment:
I specifically want these so I can refer back to the JIRA issue where I am tracking all commentary and can do an easy search through the base when I come back to an issue to address it and eventually remove them entirely. I suppose I can add the issue description to the tags to make it more clear. These are not intended to be permanent, in fact, the presence of any MYR tag/ifdef is indicative of something that must be addressed before it is released.